.github/workflows/ci-advanced.yml

```
@@ -0,0 +1,352 @@
+
+      - name: Run YAML Lint
+        run: yamllint .
+
+      - name: Run Ansible Lint
+        run: ansible-lint --force-color
+
+      - name: Run Python Lint
+        run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+
+      - name: Syntax Check
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2038:warning:1:1: Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames

(shellcheck)